tests/Command/PullRequest/PullRequestMergeCommandTest.php

```diff
@@ -94,6 +94,18 @@ class PullRequestMergeCommandTest extends CommandTestCase
 New-target: %s/%s (was "%s")
 OET;
 
+    const MERGE_NOTE_SWITCHED_BASE_AND_CLOSED = <<<OET
```

Comment:
I really don't know, just used the same heredoc delimiter that already were using the other consts. BTW, I'd like to update the usages of heredoc with nowdoc in another commit.